### DISCUSSION TITLE
Perhaps both jobs can start at the same time rather than executing one first

### DISCUSSION THREAD

**Discussion by caoyang728** [swar/Swar-Chia-Plot-Manager#777] [Ideas]
Thanks for your tools, the tools is very helpful to me. I don't know if I have a problem with the setup, 
The max_for_phase_1 in my two jobs is 6 and 5 respectively, 
Initially, he will start jobs 1's 6 tasks, and then jobs 2's 5 tasks. During jobs 1 startup, jobs 2's device is idle (it only appears when the tool is just executing)
Sorry, my english is not good, I don't konw if you understand what I mean.

**Comment by Rayrayray87** [comment]
SET minimum_minutes_between_jobs to 0 

Will solve your issue
